GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -879,7 +889,8 @@ Inputs GPURecoWorkflowSpec::inputs()
     inputs.emplace_back("tpctopologygain", gDataOriginTPC, "TOPOLOGYGAIN", 0, Lifetime::Condition, ccdbParamSpec(CDBTypeMap.at(CDBType::CalTopologyGain)));
     inputs.emplace_back("tpcthreshold", gDataOriginTPC, "PADTHRESHOLD", 0, Lifetime::Condition, ccdbParamSpec("TPC/Config/FEEPad"));
     o2::tpc::VDriftHelper::requestCCDBInputs(inputs);
-    o2::tpc::CorrectionMapsLoader::requestCCDBInputs(inputs);
+    Options optsDummy;
+    mFastTransformHelper->requestCCDBInputs(inputs, optsDummy, mSpecConfig.requireCTPLumi); // option filled here is lost
```

Comment:
not that it changes much, but you can save one line by just writing `optsDummy()` in the call to `requestCCDBI`.

Reply:
I had it like that initially, but it complained about assignment to rvalue